packages/common-integration-tests/src/dictionary.ts

```diff
@@ -1466,5 +1467,45 @@ export function runDictionaryTests(
         }
       });
     });
+
+    describe('#dictionaryLength', () => {
+      itBehavesLikeItValidates((props: ValidateDictionaryProps) => {
+        return Momento.dictionaryLength(props.cacheName, props.dictionaryName);
+      });
+
+      it('returns a miss if the dictionary does not exist', async () => {
```

Comment:
Do you know of a reproducible case to hit the code path where the length doesn't exist even if there was a hit?

Reply:
Nope, afaik dictionary cannot exist with no values in it.

Reply:
Yeah so that's the reason I asked the question above. If it's not possible for the dictionary to exist without values, is it still possible for the `length` field to be missing in the API response?